README.md

```diff
@@ -42,17 +42,6 @@ ARGUMENTS
 ---------
 <!-- TODO replace this with output from command line --help -->
 
---deposits-dir | -d (optional)
```

Comment:
Okay, this should be copied from the result of running the application with `--help` on the command line. Some things need to change:
* I propose to rename `--deposits-dir` to `--output-deposits-dir` to make it a bit more specific that this is the target of the application
* `sip-dir` should be `multi-deposit-dir`